drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -374,6 +374,14 @@ def attemps_remain(self):
     def get_user_state(self, request, suffix=''):
         """ GET all user-specific data, and any applicable feedback """
         data = self._get_user_state()
+
+        # In assessment mode, if item is placed correctly and than the page is refreshed, "correct"
+        # will spill to the frontend, making item "disabled", thus allowing students to obtain answer by trial
+        # and error + refreshing the page. In order to avoid that, we remove "correct" from an item here
+        if self.mode == self.ASSESSMENT_MODE:
+            for item_id in data['items']:
+                del data['items'][item_id]["correct"]
```

Comment:
@e-kolpakov Nice catch! I think we could just move this code into `_get_user_state`, though.